AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -423,19 +457,35 @@ am::am_Error_e CAmSocketHandler::addFDPoll(const int fd, const short event, IAmS
   */
 am_Error_e CAmSocketHandler::removeFDPoll(const sh_pollHandle_t handle)
 {
+    CHECK_CALLER_THREAD_ID()
+    
     VectorListPoll_t::iterator iterator = mListPoll.begin();
-
+    
     for (; iterator != mListPoll.end(); ++iterator)
     {
         if (iterator->handle == handle)
         {
             iterator = mListPoll.erase(iterator);
             mSetPollKeys.pollHandles.erase(handle);
             mRecreatePollfds = true;
-            return (E_OK);
+            break;
         }
     }
-    return (E_UNKNOWN);
+    
+    if (iterator == mListPoll.end())
+        return (E_UNKNOWN);
+    
+    VectorListPoll_t::iterator iteratorActivePolls = mListActivePolls.begin();
```

Comment:
Overall, if the variable is only used locally in a loop like this, I am a strong proponent of using short/simple names.  "it" would be just as readable (no, more readable) than the long and ugly "iteratorActivePolls".  In the previous loop "iterator" was considered good enough - right?  Why not here also?   Just go with "it" to be honest... it's a common convention.